### DISCUSSION TITLE
Log4j Update for operators v0.20

### DISCUSSION THREAD

**Discussion by veerendra2** [strimzi/strimzi-kafka-operator#6219] [Q&A] [answered]
Hello,
Do you guys have any planes to release to fix recent Log4j vulnerability in operators `v0.20` with Kafka version `v2.6`.

Thanks

**Comment by scholzj** [comment, marked as answer]
No, there are no plans for it. The Log4j CVEs were addressed in 0.26.1, 0.27.0 and 0.27.1 as the different CVEs were being discovered.
